src/utils/users/cpfMatches.util.ts

```diff
@@ -2,7 +2,7 @@ export const cpfMatches = {
   regex: /^\d{3}\.\d{3}\.\d{3}\-\d{2}$/gm as RegExp,
   message: {
     message: {
-      error: 'Invalid format',
+      error: 'Invalid cpf format',
```

Comment:
Uma observação aqui seria colocar o CPF em maiúsculo.

Reply:
pronto, aproveitei e padronizei de CNPJ e data de expiração, pra não precisar mexer quando a gente for usar no futuro.